www/index.html

```diff
@@ -359,6 +359,7 @@
   <script type="text/javascript" src="lib/framework7/js/framework7.min.js"></script>
   <script type="text/javascript" src="js/init-styles.js"></script>
   <script type="text/javascript" src="js/my-app.js"></script>
+  <script type="text/javascript">init();</script>
```

Comment:
Good catch! No, it is not OK, it is used elsewhere and if you search with the app, it breaks. I will fix that, thanks for pointing it out.